sass/groups/common/DeleteGroupModal.scss

```diff
@@ -0,0 +1,11 @@
+.DeleteGroupModal {
+  &__info {
+    font-size: 1.7em;
+  }
+}
+
+.DeleteGroupModal__icon {
+  span {
```

Comment:
Please do not use tag elements as selectors. This is highly inefficient (the browser selects all spans, then filters out spans that are not descendant of .DeleteGroupModal__icon).
There's also a linter rule against this, I wonder why it did not work this time :(